cloud_experts_tutorials/cloud-experts-rosa-cloudwatch-sts.adoc

```diff
@@ -1,6 +1,6 @@
 :_mod-docs-content-type: ASSEMBLY
 [id="cloud-experts-rosa-cloudwatch-sts"]
-= Tutorial: Configuring the Cluster Log Forwarder for CloudWatch logs and STS
+= Configuring log forwarding for CloudWatch logs and STS
```

Comment:
Amazon CloudWatch?

Reply:
Updated across the doc, cc @EricPonvelle to make sure that's OK

Reply:
Going to revert this change since it's actually adding additional out of scope dependencies here that are blocking merging the PR.